client/v2/example_test.go

```diff
@@ -23,8 +25,27 @@ func ExampleClient() {
 	}
 }
 
+// Create a new client with a user provided *http.Client
+func ExampleClient_HTTPCustom() {
+	c, _ := cookiejar.New(nil)
+	_, err := client.NewHTTPClient(client.HTTPConfig{
+		Addr: "http://localhost:8086",
+		// Pass in a *http.Client
+		HTTPClient: &http.Client{
+			Jar: c,
+			Transport: &http.Transport{
+				DisableKeepAlives:  true,
+				DisableCompression: true,
+			},
+		},
+	})
+	if err != nil {
+		fmt.Println("Error creating InfluxDB Client with customer HTTP client: ", err.Error())
+	}
+}
+
 // Write a point using the UDP client
-func ExampleClient_uDP() {
+func ExampleClient_UDP() {
```

Comment:
It's weird, but leaving this undercase is actually how you get godoc to parse it, it ends up being upper-case in the docs, see https://godoc.org/github.com/influxdb/influxdb/client/v2#pkg-examples